onnx/backend/test/case/node/clip.py

```diff
@@ -55,3 +77,23 @@ def export_clip_default():  # type: () -> None
         y = np.clip(x, -np.inf, 0.0)
         expect(node, inputs=[x], outputs=[y],
                name='test_clip_default_max')
+        node = onnx.helper.make_node(
+            'Clip',
+            inputs=['x'],
+            outputs=['y'],
+        )
+
+        x = np.array([-1, 0, 1]).astype(np.float32)
+        y = np.array([-1, 0, 1]).astype(np.float32)
+        expect(node, inputs=[x], outputs=[y],
+               name='test_clip_default_inbounds')
+
+        x = np.array([-1.1, 0, 1.1]).astype(np.float32)
+        y = np.array([-1.1, 0, 1.1]).astype(np.float32)
+        expect(node, inputs=[x], outputs=[y],
```

Comment:
remove this one, it's the same as the previous one, don't forget to delete the corresponding binary data. :-)